R/getWebServiceData.R

```diff
@@ -132,7 +115,7 @@ check_non_200s <- function(returnedList){
 default_ua <- function() {
```

Comment:
have I mentioned how much I like this, by the way? This approach to tagging user agents is great